packages/pds/src/oauth/provider.ts

```diff
@@ -46,12 +46,6 @@ export class PdsOAuthProvider extends OAuthProvider {
       // the PDS can use tokenId as access tokens. This allows the PDS to
       // always use up-to-date token data from the token store.
       accessTokenType: AccessTokenType.id,
-
-      onClientInfo: (clientId) => ({
-        isFirstParty: clientId === 'https://bsky.app/',
-        // @TODO make client client list configurable:
-        isTrusted: undefined,
```

Comment:
Could use a little context for this change.  If we intend to land it, also could use a changeset.

Reply:
This is currently not in use and does not match the clientId that the app should have (it should be a url to a JSON file).

Until we actually do have a clientId for our app, I figured it is better not to use this at all.

Reply:
Ah, got it! If there's no change in behavior then I am ambivalent about whether there's a changeset for it.